modules/nf-core/diamond/blastp/main.nf

```diff
@@ -4,12 +4,12 @@ process DIAMOND_BLASTP {
 
     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/diamond:2.0.15--hb97b32f_0' :
-        'biocontainers/diamond:2.0.15--hb97b32f_0' }"
+        'https://depot.galaxyproject.org/singularity/diamond:2.1.8--h43eeafb_0' :
+        'biocontainers/diamond:2.1.8--h43eeafb_0' }"
 
     input:
     tuple val(meta), path(fasta)
-    path db
+    tuple val(meta), path(db)
```

Comment:
Are you sure this is working correctly? Having the same meta name for fasta & DB seems sketchy.  
```suggestion
    tuple val(meta2), path(db)
```

Reply:
It doesn't work as you'd expect 😢 

I wish it did an implicit join.

Reply:
Changed to meta2.